### ISSUE TITLE
Server Timings is empty

### ISSUE THREAD

**Issue by FrankFromGermany** [DaxStudio/DaxStudio#85]
EVALUATE
ROW ( "Test", DISTINCTCOUNT ( FactInternetSales[OrderDate] ) )

This query takes 4ms (according to Query History) but Server Timings does not return anything!?

**Comment by marcosqlbi** [comment]
Probably the query is in cache. You should clear the cache and run the query again. You can also show the line resolved by cache in Server Timings (it is disabled by default, use the button in the ribbon to enable it).

**Comment by FrankFromGermany** [comment]
Of course I had cleared the cache! 
Maybe this issue only happens when you connect DAX Studio to Power BI.

I tried to compare

EVALUATE
 ROW ( "Test", DISTINCTCOUNT ( FactInternetSales[OrderDate] ) )

with

EVALUATE
 ROW ( "Test", CALCULATE ( COUNT ( Date[Fulldate] ), FactInternetSales)

and only for the former query Server Timings will always be empty.

Restarting both, Power BI and DAX Studio, did not help.
